cluster-http/src/test/scala/doc/akka/cluster/http/management/CompileOnlySpec.scala

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package doc.akka.cluster.http.management
+
+import akka.actor.ActorSystem
+import akka.management.cluster.ClusterHttpManagement
+
+object CompileOnlySpec {
+
+  //#loading
+  val system = ActorSystem()
+  val httpMgmt = ClusterHttpManagement(system)
```

Comment:
Is this really correct? It might work, but I thought the intended usage would be 
```
AkkaManagement(system).start()
```

which would load from reference.conf
```
http.route-providers += "akka.management.cluster.ClusterHttpManagement"
```

Reply:
That makes a great deal more sense. Would you like me to submit a PR?